lib/linter-views.coffee

```diff
@@ -110,9 +124,32 @@ class LinterViews
       )
       throw null
 
+  updateCurrentLine: (line) ->
+    if @_currentLine isnt line
+      @_currentLine = line
+      @_updateLineMessages()
+
+
+  _updateLineMessages: ->
```

Comment:
I think we need LintMessages for only current editor, also changing active pane triggers `LinterViews::render`